src/test/java/com/microsoft/sqlserver/testframework/AbstractTest.java

```diff
@@ -241,35 +243,42 @@ public static String getConfiguredProperty(String key, String defaultValue) {
     public static void invokeLogging() {
         Handler handler = null;
 
-        String enableLogging = getConfiguredProperty(Constants.MSSQL_JDBC_LOGGING, Boolean.FALSE.toString());
+        // enable logging to stream by default for tests
+        String enableLogging = getConfiguredProperty(Constants.MSSQL_JDBC_LOGGING, Boolean.TRUE.toString());
 
         // If logging is not enable then return.
         if (!Boolean.TRUE.toString().equalsIgnoreCase(enableLogging)) {
             return;
         }
 
-        String loggingHandler = getConfiguredProperty(Constants.MSSQL_JDBC_LOGGING_HANDLER, "not_configured");
+        String loggingHandler = getConfiguredProperty(Constants.MSSQL_JDBC_LOGGING_HANDLER,
+                Constants.LOGGING_HANDLER_STREAM);
 
         try {
             if (Constants.LOGGING_HANDLER_CONSOLE.equalsIgnoreCase(loggingHandler)) {
                 handler = new ConsoleHandler();
+                handler.setFormatter(new SimpleFormatter());
             } else if (Constants.LOGGING_HANDLER_FILE.equalsIgnoreCase(loggingHandler)) {
                 handler = new FileHandler(Constants.DEFAULT_DRIVER_LOG);
+                handler.setFormatter(new SimpleFormatter());
                 System.out.println("Look for Driver.log file in your classpath for detail logs");
+            } else if (Constants.LOGGING_HANDLER_STREAM.equalsIgnoreCase(loggingHandler)) {
+                handler = new StreamHandler(new PrintStream(Constants.LOGGING_STREAM), new SimpleFormatter());
```

Comment:
You should store this stream in class as a class object, initialize it here and then add to StreamHandler.
Also close it safely after all tests are done in `@AfterAll` method.